### DISCUSSION TITLE
-worldpath option doesn't seem to work

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2557] [Q&A] [answered]
Hi,

I am trying to use -worldpath on startup, but in every variation that I've tried, the server doesn't find the worlds in my Worlds folder

I've tried:
`#!/bin/sh`

`MONO_THREADS_PER_CPU=50 mono ./TShock/TerrariaServer.exe -port 7777 -maxplayers 6 -worldpath ~/Worlds/`
`MONO_THREADS_PER_CPU=50 mono ./TShock/TerrariaServer.exe -port 7777 -maxplayers 6 -worldpath "~/Worlds/"`
`MONO_THREADS_PER_CPU=50 mono ./TShock/TerrariaServer.exe -port 7777 -maxplayers 6 -worldpath "~/Worlds"`
`MONO_THREADS_PER_CPU=50 mono ./TShock/TerrariaServer.exe -port 7777 -maxplayers 6 -worldpath ~/Worlds`
`MONO_THREADS_PER_CPU=50 mono ./TShock/TerrariaServer.exe -port 7777 -maxplayers 6 -worldpath "~\\Worlds\\"`

But nothing works. Can anyone help, please?

**Comment by hakusaro** [comment, marked as answer]
Use `-worldselectpath` since `-worldpath` was removed. See: https://github.com/Pryaxis/TShock/blob/general-devel/CHANGELOG.md#tshock-440-pre-release-9
